minigotchi-ESP32/config.cpp

```diff
@@ -17,7 +18,7 @@ bool Config::deauth = true;
 bool Config::advertise = true;
 
 // screen configuration
-bool Config::display = false;
+bool Config::display = true;
```

Comment:
we should probably just leave this false by default for good measure...

Reply:
Okay.